site/content/installation/installing-nic/create-license-secret.md

```diff
@@ -0,0 +1,149 @@
+---
+docs: DOCS-602
+title: Create a License Secret 
+toc: true
+weight: 100
```

Comment:
```suggestion
title: Create a License Secret 
toc: true
weight: 100
docs: DOCS-000
```

The `docs`: value is a unique code: it should be left a placeholder for now (DocOps has automation stuff around it).